Sources/Algorithms/Keyed.swift

```diff
@@ -0,0 +1,86 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+public struct KeysAreNotUnique<Key, Element>: Error {
+  public let key: Key
+  public let previousElement: Element
+  public let conflictedElement: Element
```

Comment:
Subjective nit:  I think "conflictingElement" sounds better (more natural).

In this case, in context, it's perfectly clear either way.  The preference comes from the difference between "conflicted" and "conflicting" in a much more broad, general sense.  Technically "conflictedElement" could mean "the element that is conflicted, as in has conflicting feelings".  Nonsensical here, I know, but still it "feels" better to say "conflicting".  English is a silly language. 😆

Reply:
> "the element that is conflicted, as in has conflicting feelings"

Maybe the element is feeling a little down. Have you tried asking it? 😅  /s 

> but still it "feels" better to say "conflicting". English is a silly language. 😆

Agree!